RotationSolver/Updaters/TargetUpdater.cs

```diff
@@ -521,4 +524,28 @@ private static void UpdateNamePlate(IEnumerable<BattleChara> allTargets)
         }
         DataCenter.TreasureCharas = charas.ToArray();
     }
+
+    // status checks that can't easily be represented in config
+    private static bool CheckTargetableOther(BattleChara b) {
+        // Labyrinth of the Ancients
+        if (Svc.ClientState.TerritoryType == 174 && b.Name.ToString() == "Thanatos") {
```

Comment:
So two things here

1. Extract `"Thanatos"` to a new class and have it defined as a const, this would reduce the chances of misspelling it elsewhere 
2. Is this going to work for everyone? If I switch the client to JP for example, is it still going to be `Thanatos`?

Reply:
1. Same comment as above
2. Honestly I have no idea, I'll check on the discord to see if anyone using the client in a different language knows the answer

Reply:
Update, according to Malediktus enemy names do indeed vary by client language. I will go spelunking in the spreadsheets to see if there's some kind of unique enemy ID I can use instead

Reply:
looks like dataID does what we want